src/api/services/NotificationAPI.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * NotificationAPI Services
+ * Provide services related to Notification (Avvisatura) to communicate with PagoPaAPI
+ */
+
+import * as express from "express";
+import fetch from "node-fetch";
+import { CONFIG } from "../../Configuration";
+import { NotificationSubscriptionRequestType } from "../../enums/NotificationSubscriptionType";
+import { NotificationSubscriptionResponse } from "../types/NotificationSubscriptionResponse";
+
+// Notification Service for PagoPA communications
+export class NotificationAPI {
+  // Update subscription (Activation or Deactivation) to Notification Service for a fiscalCode
+  public static updateSubscription(
```

Comment:
it looks like this class is made only of static methods -> do not use a class, use simple functions

Reply:
as a general advice, do no use classes only as a namespace. namespaces are modules (file) in javascript / typescript.